### PR TITLE
RELATED: MKB-1716 - Add extra docs for GDUI launch

### PR DESCRIPTION
**Pull request by lupko**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/510/ : SUCCESS in 58s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/299/ : SUCCESS in 4s

File: docs/01_intro__interactive_examples.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+id: interactive_examples
+title: Interactive Code Examples
+sidebar_label: Interactive Code Examples
+copyright: (C) 2007-2020 GoodData Corporation
+---
+
+We have created a number of simple interactive examples using CodeSandbox.ui. Each example points to a CodeSandbox screen that includes a simple GoodData.UI code next to a resulting visualization.
```
Comment (no23reason) [inline docs/01_intro__interactive_examples.md:8]: The URL is codesandbox.io, not codesandbox.ui

File: docs/01_intro__why_gdui.md [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+The situation looks simple:
+-  You have data ready in your database (an application, a read replica, or even better, in a modern cloud data warehouse such as Snowflake, Redshift, or BigQuery). 
+-  There are plenty of open source charting libraries. 
+-  If you can write code, it must be easy. 
+
+It is actually not difficult to get started. If all you need is one screen with a few charts and there is nothing more urgent in your backlog, then this is not a bad way to start.
+
+However, there are a couple of things to think about:
+-  It is easy to create static charts with a few hard-coded SQL queries, but new requirements will keep coming. Are you sure your SQL queries won’t turn into a big ball of mud or a spaghetti monster?
+-  The data model may change—not that frequently, but it happens. Are you ready to refactor all the complex analytical queries when it does?
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:71]: Typo: probably missing spaces around the –

File: docs/01_intro__why_gdui.md [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+
+In various code samples earlier in this document, we were using specific visualization components such as `<BarChart />` and `<BubbleChart />`.
+
+You can imagine that your AFM enabled React library has more visualizations like that. But no visualization framework in the world will cover all visualizations you will ever need. Unless it’s extensible by design.
+
+If you think about it, the components such as `<BarChart />` or `<BubbleChart />` do two things:
+1.  They pass the AFM query to the analytical backend and retrieve the result set.
+2.  They pass the result set (maybe with some on-the-fly transformation if necessary) to the actual charting component.
+
+The first step (query execution) can be naturally separated into a reusable component. Let’s call it `<Execute />`.
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:172]: Should we link to the Execute docs?

File: docs/01_intro__why_gdui.md [hunk 4]
```diff
@@ -0,0 +1,231 @@
...
+
+And once we have this `<Execute />` component then adding it to the public API of our collection of React components is quite a natural step. You can see it in action in our [code sandbox](https://codesandbox.io/s/github/gooddata/gooddata-ui-examples/tree/master/example-execute?file=/src/App/index.js).
+
+### Integrated point-and-click development
+
+If you are still reading, you can probably write React code and you can appreciate the flexibility of a good React.js library. 
+
+On the other hand, you usually want to start with something simple. And for quick iterations over early prototypes, writing code may look like an overkill, especially if a non-technical user can create data visualization using a point-and-click user interface.
+
+Especially if the user interface is [as easy as this](https://developer.gooddata.com/assets/Images/f13aa4ea68c25b095fd77de672e82e3e.webm).
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:194]: WebM does not work in Safari.

File: docs/01_intro__why_gdui.md [hunk 5]
```diff
@@ -0,0 +1,231 @@
...
+
+You don’t want to type `<BarChart />` every time your UI designer creates a bar chart. It is more natural to grab the identifier of whatever your point-and-click colleague created. Maybe with something like this:
+
+```jsx
+<InsightView identifier={id} />
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:199]: Should we link to docs?

File: docs/01_intro__why_gdui.md [hunk 6]
```diff
@@ -0,0 +1,231 @@
...
+
+We said that we don’t want to discourage you from building your interactive end-user friendly analytics yourself. 
+
+If you decide to go that way, our key recommendation is to think twice before designing your API contract and consider the attributes-filters-measures approach rather than any shortcuts.
+
+For those of you who prefer to try something proven instead of re-inventing the infrastructure from scratch, we have decided to make the reference implementation of the aforementioned architectural best practices publicly available under the MIT open-source license.
+
+The library is called GoodData.UI, and two options are currently available from our [GitHub repository](https://github.com/gooddata/gooddata-ui-sdk):
+-  A fully open source implementation of the attributes-filters-measures querying pattern with pluggable back end.
+-  An extension of the open source library that includes Highcharts-based visualizations; since Highcharts itself is not open-source, this extension is only available under a proprietary license too. However, the source code is fully available in the YYYY GitHub repository.
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:214]: Missed placeholder "YYYY GitHub repository"

File: docs/01_intro__why_gdui.md [hunk 7]
```diff
@@ -0,0 +1,231 @@
...
+-  [Documentation and tutorials](01_intro__about_gooddataui.md).
+-  Interactive code samples.
+-  Additional examples in not so interactive but comprehensive [gallery](https://gdui-examples.herokuapp.com/).
+
+## Getting started
+
+And if you just want to get started, you can do that in just three simple steps:
+1.  Get a free GoodData account at [www.gooddata.com/free](www.gooddata.com/free) (unless you are an existing GoodData customer).
+2.  Type `npx @gooddata/create-gooddata-react-app my-app` in the terminal window.
+3.  In the newly created folder, type npm start to see the automatically generated skeleton of your first analytical web application with additional instructions.
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:226]: Should be `yarn start` with the backticks. The tool uses yarn and will blow up when run like this without yarn.

File: docs/01_intro__why_gdui.md [hunk 8]
```diff
@@ -0,0 +1,231 @@
...
+
+A complete tutorial is available from [here](02_start__using_boilerplate.md). If you prefer to use the standard create-react-app and add GoodData.UI library manually, [we have you covered too](02_start__no_boilerplate.md).
```
Comment (no23reason) [inline docs/01_intro__why_gdui.md:228]: Missing coma: covered, too

**Comment by no23reason** [comment]
Is it ok to use the docs/assets/DV_ano_video_ad.png ? It is a real product, right?

File: docs/01_intro__interactive_examples.md (hunk 1 above)
Comment (lupko) [reply docs/01_intro__interactive_examples.md]: fixed

File: docs/01_intro__why_gdui.md (hunk 2 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md]: fixed

File: docs/01_intro__why_gdui.md (hunk 3 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md]: fixed

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/511/ : SUCCESS in 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/300/ : SUCCESS in 4s

File: docs/01_intro__why_gdui.md (hunk 3 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md]: btw I have found we have one out-dated v7 doc here (50_custom__execution.md).. i've removed this from sidebar. the link from the why gdui goes to create custom visualization where the Execute component and its working is described more in depth.

File: docs/01_intro__why_gdui.md (hunk 4 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md:194]: meh; though luck :)

File: docs/01_intro__why_gdui.md (hunk 5 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md:199]: done

File: docs/01_intro__why_gdui.md (hunk 6 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md]: fixed

File: docs/01_intro__why_gdui.md (hunk 7 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md]: fixed

File: docs/01_intro__why_gdui.md (hunk 8 above)
Comment (lupko) [reply docs/01_intro__why_gdui.md]: reworded

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/513/ : SUCCESS in 1m 06s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-fossa-scanning-zuul-el7/53/ : FAILURE in 2m 54s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/145/ : SUCCESS in 21s

**Pull request merged**